plugins/legacy-plugin-chart-pivot-table/src/utils/formatCells.js

```diff
@@ -0,0 +1,41 @@
+import { formatNumber } from '@superset-ui/core';
+
+function cellFormat(i, cols, tdText, columnFormats, numberFormat, dateRegex, dateFormatter) {
```

Comment:
nit: could we rename this to `formatCellValue`? It would also be great if we could make this file TypeScript.